music21/test/testStream.py

```diff
@@ -167,24 +167,24 @@ def testMultipartStreams(self):
         s.show()
 
     def testMultipartMeasures(self):
-        '''This demonstrates obtaining slices from a stream and layering
+        '''
+        This demonstrates obtaining slices from a stream and layering
         them into individual parts.
-
-        OMIT_FROM_DOCS
-        TODO: this should show instruments
-        this is presently not showing instruments
-        probably b/c when appending to s Stream activeSite is set to that stream
         '''
         from music21 import corpus, converter
-        a = converter.parse(corpus.getWork(['mozart', 'k155', 'movement2.xml']))
-        b = a[8][4:8]
-        c = a[8][8:12]
-        d = a[8][12:16]
+        a = converter.parse(corpus.getWork(
+            workName='mozart/k155',
+            movementNumber='2',
+            fileExtensions='xml'
+        ))
+        b = a[1][0:8]
+        c = a[2][0:12]
+        d = a[3][0:16]
```

Comment:
this is an extremely old test that doesn't really follow the rules that it should -- what is a[1], a[2], a[3]?  Are they parts?  What is [0:8] etc. ?  If we can use a.parts[0], etc. this is probably much more what we'd like to have, and then call a slice from 0 to the index of measure(1) + 1, THANKS!

Reply:
I also found this recently and puzzled over it. The furthest I got was simplifying the parsing line to just:
`corpus.parse('mozart/k155', 2)`

Reply:
I believe b,c,d are the different voices Violin I, Violin II, Viola
starting at 0 allows the names to be included in the resulting stream. 
I can introspect them and try to give them better names.